### ISSUE TITLE
JUnit report format produces invalid XML for multiple testsuites

### ISSUE THREAD

**Issue by fkleon** [daveshanley/vacuum#353]
Running `report --junit $SPEC` produces an invalid XML file when the result contains failures in multiple test suites.

The XML appears to be missing the root element `testsuites` and the XML cannot be parsed by GitLab or `xmllint`:

```
 Head report parsing error: JUnit XML parsing failed: 93:1: FATAL: Extra content at the end of the document 
```

In this case line 93 is where the second `testsuite` element appears. Simplified report content below:

```xml
<testsuite name="Category: operations" tests="10" failures="10" time="0.070743857">
  <testcase name="Category: operations" classname="operation-operationId" time="0.070743857">
    [..]
  </testcase>
  [..]
</testsuite>
<!-- line 93 below -->
<testsuite name="Category: schemas" tests="1" failures="1" time="0.070743857">
  <testcase name="Category: schemas" classname="oas3-unused-component" time="0.070743857">
    [..]
  </testcase>
</testsuite>
```

**Comment by daveshanley** [comment]
Oh dear. I will look into this on the next bug run.

**Comment by daveshanley** [comment]
I added the struct for it, I just forgot to wire it up I guess.

https://github.com/daveshanley/vacuum/blob/main/vacuum-report/junit.go#L16

This is fixed in #357
